sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosSessionRetryOptions.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
+import com.azure.cosmos.models.CosmosRegionSwitchHint;
+
+/**
+ * {@link CosmosSessionRetryOptions} encapsulates hints which influence
+ * internal retry policies which are applied when the effective consistency
+ * used for the request is <i>Session Consistency</i>.
+ * */
+public final class CosmosSessionRetryOptions {
+
+    private final CosmosRegionSwitchHint regionSwitchHint;
+
+    CosmosSessionRetryOptions(CosmosRegionSwitchHint regionSwitchHint) {
+        this.regionSwitchHint = regionSwitchHint;
+    }
+
+    static void initialize() {
+        ImplementationBridgeHelpers.CosmosSessionRetryOptionsHelper.setSessionRetryOptionsAccessor(new ImplementationBridgeHelpers.CosmosSessionRetryOptionsHelper.CosmosSessionRetryOptionsAccessor() {
```

Comment:
Please follow exactly the pattern used in other classes - see CosmosClinetTelemetryConfig for example below the comment. You are missing the ctor for example (and the header config) - this is pretty brittle code in races - we have learned the hard way about a few edge cases - when multiple class loaders exists etc. - so important to just follow what finally worked elsewhere

///////////////////////////////////////////////////////////////////////////////////////////
    // the following helper/accessor only helps to access this class outside of this package.//
    ///////////////////////////////////////////////////////////////////////////////////////////

Reply:
Blocking on this comment

Reply:
Resolved in next iteration.